pysheaf/pysheaf.py

```diff
@@ -765,6 +765,38 @@ def consistencyRadii(self,assignment,testSupport=None,tol=1e-5):
                                             radii.append(self.cells[cf1.index].metric(cf1.restriction(c1.value),cf2.restriction(c2.value)))
 
         return np.unique(radii)
+    
+    def consistencyRadiusSheafCells(self,assignment_input,testSupport=None,tol=1e-5):
+        """Compute the consistency radius of an approximate section"""
```

Comment:
OK. Is this meant to be a public method? Why use this as opposed to Sheaf.consistencyRadius()?